src/main/java/org/cascadebot/cascadebot/events/PlayerListener.java

```diff
@@ -56,10 +62,18 @@ private void onEnd(AudioTrack track) {
                 }
                 // Take the next track in the queue, remove it from the queue and play it
                 AudioTrack audioTrack = player.getQueue().remove();
-                player.playTrack(audioTrack);
+                try {
+                    player.playTrack(audioTrack);
+                } catch (FriendlyException e) {
```

Comment:
Have you tested that this actually works? I was under the impression that a different thread throws this so we have to use some sort of callback with Lavaplayer.

Reply:
I don't know of a known bad song so I haven't been able to test this. I'll see if i can find one. Also I'm pretty sure the playTrack stuff is all one thread at least to the point where it would throw the exception, but I'll double check that.